implementations/rust/ockam/ockam_app/src/app/events.rs

```diff
@@ -1 +1,2 @@
 pub const SYSTEM_TRAY_ON_UPDATE: &str = "app/system_tray/on_update";
+pub const OPEN_URL: &str = "app/invite/clicked";
```

Comment:
We can also consider moving this into `src/invite/events.rs`, where it might want a comment to distinguish that it's not exactly a Tauri event in the same sense as the others. 

Otherwise I think I'd like to see the string content a little more agnostic to what clickable action the app should trigger for the user, is that possible or does it complicate things in an unproductive way?

Reply:
You adjusted this already 👍🏻